metadata.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "chocolatey-chocolatey",
-  "version": "1.2.1",
+  "name": "puppetlabs-chocolatey",
+  "version": "0.0.1",
   "author": "chocolatey",
   "summary": "The Chocolatey package provider for Puppet",
```

Comment:
Minor nit: Should we change this to say something like: 'Official Puppet provider for the Chocolatey package manager' to indicate difference between this and choco?